packages/atomic/src/utils/accessibility-utils.spec.ts

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {ReactiveControllerHost} from 'lit';
+import {describe, it, expect, vi} from 'vitest';
+import {AriaLiveRegionController} from './accessibility-utils';
+
+describe('#AriaLiveRegionController', () => {
+  let host: ReactiveControllerHost;
+  let controller: AriaLiveRegionController;
+
+  beforeEach(() => {
+    host = {
+      addController: vi.fn(),
+      removeController: vi.fn(),
+    } as unknown as ReactiveControllerHost;
+    controller = new AriaLiveRegionController(host, 'test-region', true);
+  });
+
+  it('should add itself to the host controller list on initialization', () => {
+    expect(host.addController).toHaveBeenCalledWith(controller);
+  });
+
+  describe('getAriaLiveElement', () => {
+    it('should dispatch a custom event to find the aria-live element', () => {
+      const dispatchEventSpy = vi.spyOn(document, 'dispatchEvent');
+      controller['getAriaLiveElement']();
+      expect(dispatchEventSpy).toHaveBeenCalledWith(
+        expect.objectContaining({
+          type: 'atomic/accessibility/findAriaLive',
+        })
+      );
+    });
+
+    it('should return the element from the event detail if available', () => {
+      const mockElement = {
+        updateMessage: vi.fn(),
+      } as unknown as HTMLAtomicAriaLiveElement;
+      vi.spyOn(document, 'dispatchEvent').mockImplementation((event: Event) => {
+        if (event.type === 'atomic/accessibility/findAriaLive') {
+          Object.defineProperty(event, 'detail', {
+            value: {element: mockElement},
+            writable: false,
+          });
+        }
+      });
+
+      const result = controller['getAriaLiveElement']();
+      expect(result).toBe(mockElement);
+    });
+  });
```

Comment:
We should not test private methods directly.
Instead, we should cover them through the public ones.